src/cpp/r/session/graphics/RGraphicsDevice.cpp

```diff
@@ -760,8 +762,13 @@ const double kDefaultDevicePixelRatio = 1.0;
    
 Error initialize(
             const FilePath& graphicsPath,
-            const boost::function<bool(double*,double*)>& locatorFunction)
-{      
+            const boost::function<bool(double*, double*)>& locatorFunction)
+{
+   // initialize tracing
+   std::string traceGraphicsDevice = core::system::getenv("RSTUDIO_TRACE_GRAPHICS_DEVICE");
```

Comment:
I'm ok with this change, but it seems unrelated to the rest of this PR. Just wanted to mention it in case it snuck in accidentally.

Reply:
I wanted to sneak it in just because it didn't feel worth having a whole new PR for dev-internal code.